pom.xml

```diff
@@ -89,6 +90,7 @@
               <exclude>doc/design/modules/crf.tex</exclude>
               <exclude>doc/design/modules/decision-trees.tex</exclude>
               <exclude>doc/design/modules/glm.tex</exclude>
+              <exclude>doc/design/modules/graph.tex</exclude>
```

Comment:
The license header for this file should be in the master now, there is no need to exclude it.